packages/grid/helm/helm.py

```diff
@@ -65,9 +65,9 @@ def replace_variables(d: Any) -> None:
     if "kubernetes.io/ingress.class" in d:
         d["kubernetes.io/ingress.class"] = "{{ .Values.ingress.ingressClass }}"
 
-    if "kind" in d and d["kind"] == "Ingress" and "spec" in d:
-        d["spec"]["tls"] = [{"hosts": ["{{ .Values.node.settings.hostname }}"]}]
-        d["spec"]["rules"][0]["host"] = "{{ .Values.node.settings.hostname }}"
+    # if "kind" in d and d["kind"] == "Ingress" and "spec" in d:
```

Comment:
NOTE:  This will break the staging servers.
Will try working around a nice solution for the same